packages/frontend-2/components/error/page/GenericErrorBlock.vue

```diff
@@ -27,9 +29,27 @@ import dayjs from 'dayjs'
 const SSR_DATE_STATE_KEY = 'ssr_error_page_generic_block_static_date'
 const getDate = () => new Date().toISOString()
 
-defineProps<{ error: SimpleError }>()
+const props = defineProps<{
+  error: SimpleError
+  /**
+   * We have an /error page for rendering various errors that we don't really know much about (like their
+   * actual status code), e.g. for auth errors. This makes the renderer place less of an emphasis on the status code.
+   */
+  isGenericErrorPage?: boolean
```

Comment:
puts more of an emphasis on the error and hides the status code altogether, cause its not relevant when reporting errors to `/error?message=whatever`